lib/dev/ui/interactive_prompt.rb

```diff
@@ -62,6 +57,23 @@ def call
 
       private
 
+      def reset_position
+        # This will put us back at the beginning of the options
+        # When we redraw the options, they will be overwritten
+        Dev::UI.raw do
+          num_lines = @options.join("\n").split("\n").reject(&:empty?).size
+          num_lines.times { print(ANSI.previous_line) }
+          print(ANSI.previous_line + ANSI.end_of_line + "\n")
+        end
+      end
+
+      def clear_output
+        Dev::UI.raw do
+          num_lines = @options.join("\n").split("\n").reject(&:empty?).size
```

Comment:
This logic is slightly unintuitive (I guess sometimes elements of `@options` contains `"\n"`), and it's used in two places.  Can we make this a private `num_lines` method?